package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "instamart",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node server/dist/index.js",
+    "dev": "cross-env NODE_ENV=dev ts-node-dev server/src/index.ts",
+    "build": "tsc -p .",
```

we need to make a scripts for eslint and use it in pre-commit
and we can also add the sync script for database building
```suggestion
    "build": "tsc -p .",
    "lint": "eslint --ext .ts server/src/",
    "lint:client": "eslint --ext .js .jsx client/src/",
    "lint-fix:client": "eslint --ext .js .jsx client/src/ --fix",
    "lint-fix": "eslint --ext .ts server/src/ --fix",
    "sync":"cross-env NODE_ENV=dev ts-node-dev server/src/database/sync.ts",
```